apps/voting/test/voting.js

```diff
@@ -358,17 +458,18 @@ contract('Voting App', accounts => {
         })
 
         it('creating vote as holder2 executes vote', async () => {
-            const voteId = createdVoteId(await app.newVote(EMPTY_SCRIPT, 'metadata', { from: holder2 }))
+            const voteId = createdVoteId(await app.forward(EMPTY_SCRIPT, { from: holder2 }))
             const [isOpen, isExecuted] = await app.getVote(voteId)
 
             assert.isFalse(isOpen, 'vote should be closed')
             assert.isTrue(isExecuted, 'vote should have been executed')
         })
     })
+
```

Comment:
We could add a test to check that now when creating a vote, vote is not casted.

Reply:
The test is implicit when state is checked. Because now holder31 is the creator of the vote but the initial state of the vote has 0 casted votes

Reply:
Oh, yes, you are right, it's not created by `nonHolder` anymore.